testing/grafana-image-renderer/APKBUILD

```diff
@@ -0,0 +1,39 @@
+# Contributor: Konstantin Kulikov <k.kulikov2@gmail.com>
+# Maintainer: Konstantin Kulikov <k.kulikov2@gmail.com>
+pkgname=grafana-image-renderer
+pkgver=1.0.5
+pkgrel=0
+pkgdesc="Grafana Backend Plugin that handles rendering of panels & dashboards to PNGs using headless chrome"
+url="https://github.com/grafana/grafana-image-renderer"
+arch="x86_64" # Same as grafana.
+license="Apache-2.0"
+makedepends="yarn"
+depends="nodejs ttf-opensans chromium ca-certificates"
+options="!check" # Has no test suite.
+install="$pkgname.pre-install"
+subpackages="$pkgname-openrc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/grafana/$pkgname/archive/v$pkgver.tar.gz
```

Comment:
Please don't use $pkgname in the source URL

Reply:
Done